conf/defaults.ini

```diff
@@ -560,6 +560,20 @@ facility =
 # Syslog tag. By default, the process' argv[0] is used.
 tag =
 
+[log.frontend]
+
```

Comment:
Nit, but could you drop this newline? We don't have them after section headers elsewhere in this file.